client/friends/friendlistTemplate.html

```diff
@@ -0,0 +1,5 @@
+<ul class="list no-border p-b">
+    <li class="list-item" ng-repeat="friend in friendsList" ng-click="friendClick($index)">
```

Comment:
This design is created by yuvraj . he used themeforest design which was told by you.